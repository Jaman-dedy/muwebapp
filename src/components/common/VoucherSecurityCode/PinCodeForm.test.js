import React from 'react';
import { shallow } from 'enzyme';
import PINCodeForm from '.';

describe('PinCodeForm', () => {
  it('should render correctly', () => {
    global.translate = jest.fn(() => 'text');
    const props = {
      onChange: jest.fn(),
      pinError: '',
    };
    const wrapper = shallow(<PINCodeForm {...props} />);
    expect(wrapper).toBeTruthy();
    expect(wrapper).toMatchSnapshot();
  });
});
