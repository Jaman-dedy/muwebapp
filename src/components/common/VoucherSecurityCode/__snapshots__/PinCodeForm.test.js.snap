// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinCodeForm should render correctly 1`] = `ShallowWrapper {}`;
